### ISSUE TITLE
Labels are problematic when they get too long

### ISSUE THREAD

**Issue by nop33** [alephium/alephium-wallet#159]
<img width="625" alt="image" src="https://user-images.githubusercontent.com/1579899/158421288-770c2e32-08d6-471f-a0f3-92f85b325225.png">

<img width="1312" alt="image" src="https://user-images.githubusercontent.com/1579899/158421324-d13332ba-c51b-4992-aeda-4531a5d34628.png">

**Comment by mvaivre** [comment]
Let's set a max string length. 20 seems to be a good compromise, WDYT?
We can use the input's error display to inform the user of the max size (in the address generation and address settings modals).

**Comment by nop33** [comment]
I think we can go higher than 20 because we can use ellipsis in the UI. But yeah, I wouldn't allow it to go on a second line in the input field.

**Comment by mvaivre** [comment]
Yeah, but the concept of allowing big strings to then truncate them systematically is a bit weird. I'd rather make the user think of a shorter label, if anyway they could never read the end of it :)

We could go with a fixed max value which allows us to keep our UIs nice and clean, and see if some users complain?

**Comment by nop33** [comment]
OK, let's go with that. Argument for big lables is people with big screens won't have truncated labels. Not everyone has the same screen size. And maybe the want to create hierarchical patterns within the label name, which might end up with long label names. 

Anyway, I am just trying to think of edge cases! No need to spend more time on this, I'll implement a fixed length that keeps the UI looking good on a 15' monitor, without much ellipsis (apart from maybe tables).

**Comment by mvaivre** [comment]
Sounds great! Thanks :)
